### PR TITLE
YSP-301 Card collections without images

### PR DESCRIPTION
**Pull request by laura-johnson**
### Description of work
- Adds templates for the text_card view mode for all content types.

### Functional testing steps:
- [ ] Verify text cards look as expected - identical to visual cards but with no image.

Related to: https://github.com/yalesites-org/yalesites-project/pull/600

### CODE REVIEW

**Review by codechefmarc — APPROVED** [review]
Tested over on the multidev on the main repo. Works great. Approved! May want to have @joetower take a quick look too.

**Comment by nJim** [comment]
Marking this work with the label "Don't Merge" as this PR should not be merged until the platform work is tested and approved. https://github.com/yalesites-org/yalesites-project/pull/600

**Pull request closed without merging**
